src/map/mercenary.h

```diff
@@ -71,6 +71,16 @@ enum merc_id {
 	MERID_MER_SWORDMAN10 = 6046, ///< MER_SWORDMAN10 / Wayne
 };
 
+enum e_merc_delete_type {
+	MDT_EXPIRED = 0,
+	MDT_KILLED = 1,
+	MDT_REMOVED = 2,
+	MDT_RANAWAY = 3,
+#ifndef MDT_MAX
+	MDT_MAX = 3,
+#endif
```

Comment:
not too sure about this, but this MAX doesn't sound right

I compare with existing ones in our emulator ... see how they do it
```
enum script_mercinfo_types {
	MERCINFO_ID = 0,
	MERCINFO_CLASS,
	MERCINFO_NAME,
	MERCINFO_FAITH,
	MERCINFO_CALLS,
	MERCINFO_KILLCOUNT,
	MERCINFO_LIFETIME,
	MERCINFO_LEVEL,
	MERCINFO_GID,

	MERCINFO_MAX
};
```
```
	UDT_MERC_KILLCOUNT,
	UDT_STATPOINT,
	UDT_ROBE,
	UDT_BODY2,
	UDT_GROUP,
	UDT_DAMAGE_TAKEN_RATE,
	UDT_MAX
};
```
```
if (type < UDT_TYPE || type >= UDT_MAX) {
```

Reply:
MDT_MAX should be 4